README.md

```diff
@@ -7,7 +7,9 @@ How we do reviews in Guest Experience:
 3. Create a pull-request as soon as you like.
 4. Do whatever you want with it. Squash and rebase to your heart's content. 
 5. Get reviewed.
-6. Just keep squashing and rebasing.
+6. Add more commits. Squash and rebase *those* as much as you like, but don't
```

Comment:
Note that I can now review the single updated commit individually.

I can also go back and review the whole set of changes if I like.

![review](https://user-images.githubusercontent.com/2403023/27233435-72553752-5287-11e7-9089-8879600fd20d.png)